extensions/windows-event-log/ConsumeWindowsEventLog.cpp

```diff
@@ -173,6 +174,13 @@ const core::Property ConsumeWindowsEventLog::ProcessOldEvents(
   withDescription("This property defines if old events (which are created before first time server is started) should be processed.")->
   build());
 
+const core::Property ConsumeWindowsEventLog::CacheUsernameLookups(
+    core::PropertyBuilder::createProperty("Cache Username Lookups")->
```

Comment:
I would rename this to identifier lookups or SID lookups, because SIDs can represent usernames and groups alike.

from [Microsoft docs](https://learn.microsoft.com/en-us/windows-server/identity/ad-ds/manage/understand-security-identifiers):
> A security identifier is used to uniquely identify a security principal or security group. Security principals can represent any entity that can be authenticated by the operating system, such as a user account, a computer account, or a thread or process that runs in the security context of a user or computer account.

Reply:
done in b56d8c21c498ba6dad859e1b450b0028dfdbc002